recipe/meta.yaml

```diff
@@ -1,45 +1,54 @@
 {% set name = "streamlit-aggrid" %}
-{% set version = "0.3.3" %}
+{% set version = "0.3.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d96edb102df98450f3b6bcf5a52b64e6ceef9678e82e2dba9cfef074e7f134ab
+  sha256: d4b776e4e7c2a91808744873843c8236065e263c22cc39e7267383ccc81486a3
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  # poetry not available for py312
+  # streamlit not available for s390x
+  skip: true  # [py<38 or py>311 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
     - poetry
-    - python >=3.7,<4.0
+    - python
   run:
+    - python !=3.9.7
```

Comment:
Where does this come from? I'm not too sure to see the provenance of this.

Reply:
https://github.com/PablocFonseca/streamlit-aggrid/blob/e673472b9298cc62f212466404d9256a6a020cb5/pyproject.toml#L14

Reply:
That's 0.3.5.

Reply:
I will restore it
https://inspector.pypi.io/project/streamlit-aggrid/0.3.4